src/htmx.js

```diff
@@ -4666,9 +4668,6 @@ var htmx = (function() {
         saveCurrentPageToHistory()
       }
 
-      if (hasHeader(xhr, /HX-Reswap:/i)) {
-        swapOverride = xhr.getResponseHeader('HX-Reswap')
-      }
```

Comment:
Is the `HX-Reswap` header being removed here?

Reply:
Not sure what you mean, but I removed that bit because it would override the decision of the event and the same thing has already happened in line 4627 anyway. So the header is still honored, just the event comes after it and can still change the swap style. That order makes sense to me.